frontend/src/components/VerticalWord/CrosswordBoxContainer.tsx

```diff
@@ -47,15 +63,20 @@ const CrosswordBoxContainer = ({ value }: CrosswordBoxContainerProps) => {
       (event.currentTarget.nextSibling as HTMLElement).focus();
     }
   };
+
   return (
     <CrosswordContainer>
-      {value.split('').map((_, i) => (
-        <CrosswordInputBox
-          key={i}
-          onInput={(event) => crosswordBoxInputHandler(event, i)}
-          onDelete={(event) => keyStrokeHandler(event, i)}
-        />
-      ))}
+      {grid.map( (row, i) => 
+        <CrosswordRow>
+          {row.map( (column, j) => 
+            <CrosswordInputBox
+              key={i + j}

```

Comment:
Key I+j will occasionally create conflicts leading to warnings in the console. Could do a string ‘x:${i}, y:${j}’

Reply:
fixed by changing to i*dimension + j